README.md

```diff
@@ -99,7 +100,12 @@ cd catkin_ws
 source devel/setup.bash
 ```
 
-Now you can launch the software. 
+Now you can launch the bringup package that brings up the launch file for each package:
+
+```bash
+roslaunch marimbabot_bringup marimbabot.launch
+```
+
```

Comment:
Maybe also add the vision launch file

Reply:
Do you want the marimbabot_bringup launch file to include all other .launch files?
Or what is the purpose of bringup in your opinion?

Reply:
Normally each package e.g. vision have their own launch files in their "launch" folder.
The bringup launches all components and gets the full system up and running.